shopify_python/google_styleguide.py

```diff
@@ -228,3 +237,15 @@ def __use_simple_list_comp(self, node):  # type: (astroid.ListComp) -> None
         """List comprehensions are okay to use for simple cases."""
         if len(node.generators) > 1:
             self.add_message('complex-list-comp', node=node)
+
+    def __use_cond_expr(self, node):  # type: (astroid.If) -> None
+        """Only one liner conditional expressions"""
+
+        if (len(node.body) == 1) and (len(node.orelse) == 1):
+            try:
+                if_body_name = node.body[0].targets[0].name
+                else_body_name = node.orelse[0].targets[0].name
+            except AttributeError:
```

Comment:
It's unclear which attribute might be missing here. Could you use a `hasattr` check instead?